content_sync/pipelines/definitions/concourse/mass-publish.yml

```diff
@@ -145,7 +145,7 @@ jobs:
           - -X
           - POST
           - -H
-          - 'Fastly-Key: ((fastly.api_token))'
+          - 'Fastly-Key: ((fastly_((version)).api_token))'
```

Comment:
Is this going to work? It's a nested vars setting, so will it pick up the internal var when setting the pipeline? We can change the path for the values in Vault if that's easier, it just needs to map to something that fits with the lookup paths that Concourse uses: https://concourse-ci.org/vault-credential-manager.html#vault-credential-lookup-rules

Reply:
@blarghmatey Where are `fastly_live` and `fastly_draft` mapped to now? If it's under `/concourse/ocw/fastly_live` I'd expect it to work, but if it's under `/concourse/ocw/ocw-theme-assets/fastly_live` then it won't.

Reply:
Tried a truncated version of the pipeline (fastly step only) here: https://cicd-qa.odl.mit.edu/teams/ocw/pipelines/mbfastlytest/jobs/mass-publish/builds/1?vars.version=%22live%22

Reply:
It's under `/concourse/ocw/fastly_live` currently

Reply:
It seems to be working based on the truncated test pipeline I created today.